src/dbnode/storage/shard.go

```diff
@@ -1872,6 +1872,11 @@ func (s *dbShard) Bootstrap(
 		}
 
 		s.markWarmFlushStateSuccess(at)
+		// Cold version needs to get bootstrapped so that the 1:1 relationship between volume number
+		// and cold version is maintained and the volume numbers / flush versions remaining monotonically
+		// increasing.
+		// TODO(rartoul): Add a test for this before merging.
```

Comment:
super nit: Want to use the TODO_COLD_FLUSH... flag?

Reply:
Oh before merging, never mind.

Reply:
added the coverage